server/src/main/java/org/cloudfoundry/identity/uaa/zone/GeneralIdentityZoneConfigurationValidator.java

```diff
@@ -83,6 +91,18 @@ public IdentityZoneConfiguration validate(IdentityZone zone, IdentityZoneValidat
         return config;
     }
 
+    private void validateRegexStrings(List<String> uris, String fieldName) throws InvalidIdentityZoneConfigurationException {
+        if (uris != null) {
+            for (String uri : uris) {
+                try {
+                    Pattern.compile(uri);
```

Comment:
sonar suggests that you do Pattern.compile(Pattern.quote(uri));

Reply:
We tried doing that change but tests started failing after that. Since this is complaining about a value that is not a user input, what we did was ignore the specific validation for this method.

Reply:
Yeah, the tests probably require that these strings actually work with regex metacharacters, which is what Sonar is complaining about.